cmd/orb-server/startcmd/start.go

```
@@ -1800,3 +1809,26 @@ func asURIs(strs ...string) ([]*url.URL, error) {
 
 	return uris, nil
 }
+
+func NewMetrics(parameters *orbParameters) (metricsProvider.Metrics, error) {
```

An error is never returned.